### PR TITLE
v7.3.1 Preview Release preparations

### PR DESCRIPTION
**Pull request by cheenamalhotra**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,31 @@ All notable changes to this project will be documented in this file.
...
 The format is based on [Keep a Changelog](http://keepachangelog.com/) 
 
+## [7.3.1] Preview Release
+### Added
+- Added support for NTLM Authentication [#998](https://github.com/Microsoft/mssql-jdbc/pull/998)
+- Added new connection property 'useFmtOnly' to retrieve parameter metadata [#1044](https://github.com/Microsoft/mssql-jdbc/pull/1044)
+- Added support for JDK 12 and build additional "jre12" JAR [#1050](https://github.com/Microsoft/mssql-jdbc/pull/1050)
+- Added 'keyVaultProviderClientId' and 'keyVaultProviderClientKey' connection properties to enhance Always Encrypted usability [#902](https://github.com/Microsoft/mssql-jdbc/pull/902)
+- Implemented `hashCode()` and `equals()` APIs for `SQLServerDataTable` and `SQLServerDataColumn` [#1025](https://github.com/Microsoft/mssql-jdbc/pull/1025)
+- Added Maven shade plugin configuration to package the driver in uber-jar [#1043](https://github.com/Microsoft/mssql-jdbc/pull/1043) [#1078](https://github.com/Microsoft/mssql-jdbc/pull/1078) [#1081](https://github.com/Microsoft/mssql-jdbc/pull/1081)
```
Comment (rene-ye) [inline CHANGELOG.md]: package the driver into uber-jar(s)?

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,31 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/) 
 
+## [7.3.1] Preview Release
+### Added
+- Added support for NTLM Authentication [#998](https://github.com/Microsoft/mssql-jdbc/pull/998)
+- Added new connection property 'useFmtOnly' to retrieve parameter metadata [#1044](https://github.com/Microsoft/mssql-jdbc/pull/1044)
+- Added support for JDK 12 and build additional "jre12" JAR [#1050](https://github.com/Microsoft/mssql-jdbc/pull/1050)
+- Added 'keyVaultProviderClientId' and 'keyVaultProviderClientKey' connection properties to enhance Always Encrypted usability [#902](https://github.com/Microsoft/mssql-jdbc/pull/902)
```
Comment (peterbae) [inline CHANGELOG.md]: Did we mean to use ` here instead of '?

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,31 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/) 
 
+## [7.3.1] Preview Release
+### Added
+- Added support for NTLM Authentication [#998](https://github.com/Microsoft/mssql-jdbc/pull/998)
+- Added new connection property 'useFmtOnly' to retrieve parameter metadata [#1044](https://github.com/Microsoft/mssql-jdbc/pull/1044)
```
Comment (peterbae) [inline CHANGELOG.md]: 'useFmtOnly' => `useFmtOnly`?

File: README.md [hunk 4]
```diff
@@ -35,27 +35,29 @@ What's coming next?  We will look into adding a more comprehensive set of tests,
 * [**Windows + SQL Server + Java**](https://www.microsoft.com/en-us/sql-server/developer-get-started/java/windows)
 
 ## Build
-### Prerequisites
-* Java 11
+### Pre-requisites
```
Comment (peterbae) [inline README.md]: I think prerequisite is a legitimate word

File: CHANGELOG.md (hunk 3 above)
Comment (cheenamalhotra) [reply CHANGELOG.md]: UseFmtOnly is connection property (a string) not a source code API. ` is only meant for Code insertion, e.g. driver methods/class names/APIs

File: CHANGELOG.md (hunk 1 above)
Comment (lilgreenbird) [reply CHANGELOG.md]: I thought we're using the term "shaded"?

Comment (cheenamalhotra) [reply CHANGELOG.md]: Shaded is jar suffix, right terminology of these jars is "uber jars" or "fat jars"

File: README.md [hunk 5]
```diff
@@ -201,36 +215,40 @@ Thank you!
...
+Starting with 6.0, stable versions have an even minor version. For example, 6.0, 6.2, 6.4, 7.0, 7.2. Preview versions have an odd minor version. For example, 6.1, 6.3, 6.5, 7.1, 7.3, and so on
 
 ## Contributors 
-Special thanks to everyone who has contributed to the project.
+Special thanks to everyone who has contributed to the project. 
 
 Up-to-date list of contributors: https://github.com/Microsoft/mssql-jdbc/graphs/contributors
 
-- marschall (Philippe Marschall)
+Here are our Top 15 contributors from community:
```
Comment (rene-ye) [inline README.md]: from the community

File: CHANGELOG.md [hunk 6]
```diff
@@ -3,6 +3,31 @@ All notable changes to this project will be documented in this file.
...
+- Added support for JDK 12 and build additional "jre12" JAR [#1050](https://github.com/Microsoft/mssql-jdbc/pull/1050)
+- Added 'keyVaultProviderClientId' and 'keyVaultProviderClientKey' connection properties to enhance Always Encrypted usability [#902](https://github.com/Microsoft/mssql-jdbc/pull/902)
+- Implemented `hashCode()` and `equals()` APIs for `SQLServerDataTable` and `SQLServerDataColumn` [#1025](https://github.com/Microsoft/mssql-jdbc/pull/1025)
+- Added Maven shade plugin configuration to package the driver jars in uber-jars [#1043](https://github.com/Microsoft/mssql-jdbc/pull/1043) [#1078](https://github.com/Microsoft/mssql-jdbc/pull/1078) [#1081](https://github.com/Microsoft/mssql-jdbc/pull/1081)
```
Comment (ulvii) [inline CHANGELOG.md]: Shade Plugin

File: CHANGELOG.md [hunk 7]
```diff
@@ -3,6 +3,31 @@ All notable changes to this project will be documented in this file.
...
+
+### Fixed Issues
+- Fixed `DatabaseMetadata.getColumns()` API to return `ResultSet` as per JDBC 4.3 Specifications [#1016](https://github.com/Microsoft/mssql-jdbc/pull/1016)
+- Fixed issue with invalid Spatial datatypes by marking them valid by default [#1035](https://github.com/microsoft/mssql-jdbc/pull/1035)
```
Comment (ulvii) [inline CHANGELOG.md]: "data type" instead of "datatype"

File: CHANGELOG.md [hunk 8]
```diff
@@ -3,6 +3,31 @@ All notable changes to this project will be documented in this file.
...
+- Fixed misleading exception message in `SQLServerCallableStatement` implementation [#1064](https://github.com/Microsoft/mssql-jdbc/pull/1064)
+- JUnit Test fixes and improvements [#1008](https://github.com/Microsoft/mssql-jdbc/pull/1008) [#1004](https://github.com/Microsoft/mssql-jdbc/pull/1004) [#994](https://github.com/Microsoft/mssql-jdbc/pull/994) [#1006](https://github.com/Microsoft/mssql-jdbc/pull/1006) [#1019](https://github.com/Microsoft/mssql-jdbc/pull/1019) [#1017](https://github.com/Microsoft/mssql-jdbc/pull/1017) [#1005](https://github.com/Microsoft/mssql-jdbc/pull/1005) [#1015](https://github.com/Microsoft/mssql-jdbc/pull/1015) [#1032](https://github.com/Microsoft/mssql-jdbc/pull/1032) [#1036](https://github.com/Microsoft/mssql-jdbc/pull/1036) [#1041](https://github.com/Microsoft/mssql-jdbc/pull/1041) [#1027](https://github.com/Microsoft/mssql-jdbc/pull/1027) [#1047](https://github.com/Microsoft/mssql-jdbc/pull/1047) [#1060](https://github.com/Microsoft/mssql-jdbc/pull/1060) [#1034](https://github.com/Microsoft/mssql-jdbc/pull/1034)
```
Comment (ulvii) [inline CHANGELOG.md]: Would look nicer if these are ordered.

File: README.md [hunk 9]
```diff
@@ -201,36 +215,40 @@ Thank you!
 ### Reporting security issues and security bugs
 Security issues and bugs should be reported privately, via email, to the Microsoft Security Response Center (MSRC) [secure@microsoft.com](mailto:secure@microsoft.com). You should receive a response within 24 hours. If for some reason you do not, please follow up via email to ensure we received your original message. Further information, including the MSRC PGP key, can be found in the [Security TechCenter](https://technet.microsoft.com/en-us/security/ff852094.aspx).
 
-## Release roadmap and standards
+## Release road-map and standards
```
Comment (ulvii) [inline README.md]: I don't think "road-map" is a better choice. Please use either "roadmap" or "road map".

File: README.md [hunk 10]
```diff
@@ -21,9 +21,9 @@ Let us know how you think we're doing.
 <a href="https://aka.ms/mssqljdbcsurvey"><img style="float: right;"  height="67" width="156" src="https://sqlchoice.blob.core.windows.net/sqlchoice/static/images/survey.png"></a>
 
 ## Status of Most Recent Builds
-| Azure Pipelines (Windows)       | Travis CI (Linux) |
+| Azure Pipelines (Windows) | Azure Pipelines (Linux) | Azure Pipelines (MacOS) |
 |--------------------------|--------------------------|
```
Comment (ulvii) [inline README.md]: Please click [View file](https://github.com/microsoft/mssql-jdbc/blob/abd4a2b94184a627011f8df333f55ee7a5a7c2ee/README.md), the table seems to be broken.

File: README.md [hunk 11]
```diff
@@ -21,9 +21,9 @@ Let us know how you think we're doing.
 <a href="https://aka.ms/mssqljdbcsurvey"><img style="float: right;"  height="67" width="156" src="https://sqlchoice.blob.core.windows.net/sqlchoice/static/images/survey.png"></a>
 
 ## Status of Most Recent Builds
-| Azure Pipelines (Windows)       | Travis CI (Linux) |
+| Azure Pipelines (Windows) | Azure Pipelines (Linux) | Azure Pipelines (MacOS) |
```
Comment (ulvii) [inline README.md]: Please click [View file](https://github.com/microsoft/mssql-jdbc/blob/abd4a2b94184a627011f8df333f55ee7a5a7c2ee/README.md), the table seems to be broken.

**Pull request merged**